app/screens/Table/components/Species.js

```diff
@@ -146,9 +146,9 @@ class Species extends React.Component {
       let extraPokemonNeededSpan
 
       if (specie.evolves > 0) {
-        const totalCandyNeeded = specie.candyToEvolve * specie.count
+        const totalCandyNeeded = (specie.candyToEvolve - 1) * specie.count + 1
         const extraCandyNeeded = totalCandyNeeded - specie.candy
-        const extraPokemonNeeded = Math.floor((extraCandyNeeded * -1) / specie.candyToEvolve)
+        const extraPokemonNeeded = Math.floor((extraCandyNeeded * -1) / (specie.candyToEvolve - 1))
 
```

Comment:
Here's a screenshot on Lumi with the calculations. You can take a look at it to see how the changes would look like:

![captura de pantalla 2017-03-14 a las 0 12 34](https://cloud.githubusercontent.com/assets/1107521/23879045/1380bcaa-084b-11e7-82ab-94e13d3070b3.png)

Here's the calculations in case you want to copy/paste them somewhere.

```
Pidgey
candyToEvolve = 12 = 12
count = 2 = 2
candy = 23 = 23

oldTotalCandyNeeded = candytoEvolve * count = 800
newTotalCandyNeeded = (candyToEvolve - 1) * count + 1 = 799
oldExtraCandyNeeded = oldTotalCandyNeeded - candy = 1
newExtraCandyNeeded = newTotalCandyNeeded - candy = 0
oldExtraPokemonNeeded = ((oldExtraCandyNeeded * -1) / (candyToEvolve - 1)) = -0
newExtraPokemonNeeded = ((newExtraCandyNeeded * -1) / (candyToEvolve - 1))
```